index.bs

```diff
@@ -1462,3 +1484,231 @@ The [=remote end steps=] with |command parameters| are:
     1. Return the result of [=updating the event map=] with [=current session=],
        |list of event names|, |list of contexts| and enabled false.
 </div>
+
+
+## browsingContext ## {#module-browsingContext}
+
+The <dfn export for=modules>Browsing Context</dfn> module contains commands and
+events relating to browsing contexts.
+
+### Definition ### {#module-browsingContext-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+BrowsingContextCommand = (BrowsingContextGetAllCommand)
+</pre>
+
+[=local end definition=]
+
+<pre class="cddl local-cddl">
+
+BrowsingContextResult = (BrowsingContextGetAllResult)
+
+BrowsingContextEvent = (
+    BrowsingContextCreatedEvent //
+    BrowsingContextDestroyedEvent
+)
+
+BrowsingContextInfo = {
+  context: BrowsingContext,
+  parent: BrowsingContext / null,
+  url: text,
+}
+
+</pre>
+
+### Types ### {#module-browsingcontext-types}
+
+#### BrowsingContext #### {#types-browsingcontext}
+
+[=remote end definition=] and [=local end definition=]
+
+<pre class="cddl remote-cddl local-cddl">
+BrowsingContext = text;
+</pre>
+
+Each [=/browsing context=] has an associated <dfn export>browsing context
+id</dfn>, which is a string uniquely identifying that browsing context. This is
+implicitly set when the context is created. For browsing contexts with an
+associated WebDriver [=window handle=] the [=/browsing context id=] must be the
+same as the [=window handle=].
+
+<div algorithm>
+To <dfn>get a browsing context</dfn> given |context id|:
+
+ 1. If |context id| is null, return [=success=] with data null.
+
+ 1. If there is no browsing context with [=browsing context id=] |context id| return
+    [=error=] with [=error code=] [=no such frame=]
+
+ 1. Let |context| be the browsing context with id |context id|.
+
+ 1. Return [=success=] with data |context|
+
+</div>
+
+<div algorithm>
+To <dfn>get the browsing context info</dfn> given |context|:
+ 1. Let |context id| be the [=browsing context id=] for |context|.
+
+ 1. If |context| has a [=parent browsing context=] let |parent id| be the
+    [=browsing context id=] of that parent. Otherwise let |parent id| be null.
+
+ 1. Let |document| be |context|'s [=active document=].
+
+ 1. Let |url| be the result of running the [=URL serializer=], given
```

Comment:
Can we be explicit about whether or not this should include any fragment? https://chromedevtools.github.io/devtools-protocol/tot/Page/#type-Frame has variants for both, so it would be easy to accidentally use the wrong thing when implementing over CDP.

Reply:
Note added was "This includes the fragment component of the URL", thanks!